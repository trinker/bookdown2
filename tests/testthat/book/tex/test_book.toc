\contentsline {chapter}{Contents}{iii}{section*.1}
\contentsline {chapter}{\chapternumberline {1}Heading of Test Chapter 1}{1}{chapter.1}
\contentsline {section}{References}{2}{section*.2}
\contentsline {chapter}{\chapternumberline {2}Heading of Test Chapter 2}{3}{chapter.2}
\contentsline {section}{References}{4}{section*.3}
