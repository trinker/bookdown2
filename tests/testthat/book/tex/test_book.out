\BOOKMARK [0][-]{section*.1}{Contents}{}% 1
\BOOKMARK [0][-]{chapter.1}{Heading\040of\040Test\040Chapter\0401}{}% 2
\BOOKMARK [1][-]{section*.2}{References}{chapter.1}% 3
\BOOKMARK [0][-]{chapter.2}{Heading\040of\040Test\040Chapter\0402}{}% 4
\BOOKMARK [1][-]{section*.3}{References}{chapter.2}% 5
